lib/assessment.js

```diff
@@ -13,14 +14,20 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var result = 0;
+  for (var i=0; i<arguments.length; i++){
+    result += arguments[i];
+  }return result;
 };
```

Comment:
Awesome. This should work great.

In the future, for clarity purposes, unless you're following an `if` statement's closing curly brace with an `else`, let closing curly braces have the whole line.